src/components/Attachment/Attachment.jsx

```diff
@@ -167,10 +169,12 @@ const Wrapper = styled(Flex)`
   ${space}
 `
 
-const AttachmentIcon = styled(Icon)`
+const AttachmentIcon = styled(Icon)(
+  ({ theme: { colors } }) => `
```

Comment:
Colocar o helper css do styled-components
```suggestion
  ({ theme: { colors } }) => css`
```

Teoricamente o css se torna obrigatório quando rola alguma interpolaçao no meio do template string, porém acho válido colocar nesse caso também para manter o padrao.

**doc** do helper css: https://styled-components.com/docs/api#css

concorda @igorcasconi?

Reply:
Sim é uma boa utilizar o css quando está usando props desestruturadas no styled components.